src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/investigate-mobile-app-crash-report.mdx

```diff
@@ -60,21 +60,10 @@ Use any of New Relic's [standard UI functions](/docs/data-analysis/user-interfac
   </Collapser>
 
   <Collapser
-    id="interaction-trail"
-    title="Interaction trail"
+    id="Event-trail"
+    title="Event trail"
```

Comment:
Should this section be removed and replace with info on the Event trail feature?

Reply:
See last comment, but yes. I think that is a good idea. I think we are opting to push the user to use the Event trail instead.

Reply:
1. The event trail shows all event types leading up to a crash in chronological order paired with their timestamp. 
2. You can sort the event trail chronologically in both directions, though it defaults to showing the oldest event first, usually an App launch. Each of the events is a collapsable component, opening the accordion will give the user more information about that event. For example, if the event is a request, then the user might find responseTime and requestUrl in the open accordion. Finally, the user can filter the event trail by clicking the icons in the legend at the top of the event trail component. Clicking an icon will remove it from the event trail.
3. The user can get a sense of what led up to a crash by using this event trail. Like, a crash happens and now I know that the user did x, y, and then z before the crash happened.

There is a current page about the event trail here - https://docs.newrelic.com/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail/. It should work the same way. But the part about the interaction trail at the bottom should go.

Reply:
Thank you for this - super helpful! I updated the collapser to reflect what you provided.